Libraries/PeriphDrivers/Source/TMR/tmr_revb.c

```diff
@@ -443,7 +443,7 @@ int MXC_TMR_RevB_GetTime(mxc_tmr_revb_regs_t *tmr, uint32_t ticks, uint32_t *tim
     uint32_t prescale = (tmr->ctrl0 & MXC_F_TMR_REVB_CTRL0_CLKDIV_A) >>
                         MXC_F_TMR_REVB_CTRL0_CLKDIV_A_POS;
 
-    uint32_t timerClock = MXC_TMR_RevB_GetClockSourceFreq(tmr);
+    int timerClock = MXC_TMR_RevB_GetClockSourceFreq(tmr);
```

Comment:
Make this an `unsigned int` since we cannot have negative clock frequencies.  This will change the return type of all the getters as well.

Reply:
Nevermind - we check this for error codes.

Just update the function definitions to use `unsigned int`

Reply:
Nevermind - we use the negative codes for errors